src/view/com/composer/Composer.tsx

```diff
@@ -655,17 +704,17 @@ function ComposerPost({
   const onPhotoPasted = useCallback(
     async (uri: string) => {
       if (uri.startsWith('data:video/')) {
-        onSelectVideo({uri, type: 'video', height: 0, width: 0})
+        onSelectVideo(post.id, {uri, type: 'video', height: 0, width: 0})
       } else {
         const res = await pasteImage(uri)
         onImageAdd([res])
       }
     },
-    [onSelectVideo, onImageAdd],
+    [post.id, onSelectVideo, onImageAdd],
   )
 
   return (
-    <>
+    <View style={[styles.post, !isActive && styles.inactivePost]}>
```

Comment:
Just leaving a note for reference:

If we encounter problems with being able to scroll on Android over "disabled" post areas, I think the solution would be to actually set the text inputs as not-editable, essentially disabling them and preventing "interception" of touches.

Unclear though if that was just an emulator issue or a bigger problem. We'll know more once testing on physical device.